doc/api/util.md

```diff
@@ -68,8 +68,7 @@ exports.puts = util.deprecate(() => {
 It returns a modified function which warns once by default.
 
 If `--no-deprecation` is set then this function is a NO-OP.  Configurable
-at run-time through the `process.noDeprecation` boolean (only effective
-when set before a module is loaded.)
+at run-time through the `process.noDeprecation` boolean.
```

Comment:
I think the original note should be left but perhaps clarified. Setting this will only work if you do it before deprecations are emitted.